client/my-sites/plan-features-2023-grid/style.scss

```diff
@@ -699,18 +706,18 @@ body.is-section-signup.is-white-signup .is-onboarding-2023-pricing-grid .signup_
 		padding: 6px 11px;
 	}
 
-	.segmented-control__item {
+	li.segmented-control__item:not(.is-selected) {
 		border: 6px;
 		padding: 2px;
 		font-weight: 500;
 
-		& .segmented-control__link {
+		& a.segmented-control__link {
 			border: 1px solid #f2f2f2;
 			border-radius: 5px; /* stylelint-disable-line scales/radii */
 
 			&:hover {
 				border-color: rgba(0, 0, 0, 0.04);
-				background: rgba(255, 255, 255, 0.3);
+				background: rgba(255, 255, 255, 0.3) !important;
```

Comment:
As a general principle, I wish we could avoid the use of `!important`, as this cannot be overridden in any other way now.

Reply:
I agree, but in this case there was no other way since there's already an `!important` defined for it [on this line](https://github.com/Automattic/wp-calypso/blob/trunk/client/landing/stepper/declarative-flow/internals/steps-repository/plans/style.scss#L158).

Reply:
oh well 🤷

Reply:
@chriskmnds The other day, I actually started to explore this and I even went another level up and saw another `!important`. And I decided to keep quiet 😭 .